pkgs/tools/networking/tinc/pre.nix

```
@@ -2,12 +2,12 @@
 
 stdenv.mkDerivation rec {
   pname = "tinc";
-  version = "1.1pre17";
+  version = "1.1pre18";
 
   src = fetchgit {
     rev = "refs/tags/release-${version}";
     url = "git://tinc-vpn.org/tinc";
```

works with proxies and is encrypted:

```suggestion
    url = "https://tinc-vpn.org/git/tinc";
```